packages/headless/src/insight.index.ts

```diff
@@ -30,11 +30,89 @@ export type {
 } from './controllers/controller/headless-controller';
 export {buildController} from './controllers/controller/headless-controller';
 
-export type {InsightInterfaceState} from './features/insight-interface/insight-interface-state';
-export type {InsightInterface} from './controllers/insight-interface/insight-interface';
-export {buildInsightInterface} from './controllers/insight-interface/insight-interface';
 export type {
-  InsightSearchBox,
+  CategoryFacetOptions,
+  CategoryFacetSearchOptions,
+  CategoryFacetProps,
+  CategoryFacetState,
+  CategoryFacet,
+  CategoryFacetValue,
+  CategoryFacetSearch,
+  CategoryFacetSearchState,
+  CategoryFacetSearchResult,
+  CoreCategoryFacet,
+  CoreCategoryFacetState,
+} from './controllers/insight/facets/category-facet/headless-insight-category-facet';
+export {buildCategoryFacet} from './controllers/insight/facets/category-facet/headless-insight-category-facet';
+
+export type {
+  QuerySummaryState,
+  QuerySummary,
+} from './controllers/insight/query-summary/headless-insight-query-summary';
+export {buildQuerySummary} from './controllers/insight/query-summary/headless-insight-query-summary';
+export {buildCoreQuerySummary} from './controllers/core/query-summary/headless-core-query-summary';
```

Comment:
Export core?

Reply:
I don't think we need to export the `buildCoreQuerySummary` we already export the `buildQuerySummary` from the insight controllers.

Reply:
sorry was commenting on an old version of the file xD